python/jupyter_leaflet/src/controls/GeomanDrawControl.ts

```diff
@@ -57,26 +60,32 @@ export class LeafletGeomanDrawControlView extends LeafletControlView {
         }
       },
       pointToLayer: function (feature, latlng) {
-        var options;
-        if (feature.properties.style.textMarker) {
+        let options;
+        if (feature?.properties?.style?.textMarker) {
           options = {
             textMarker: feature.properties.style.textMarker,
             text: feature.properties.style.text,
           };
         } else {
-          options = feature.properties.options;
+          options = feature.properties?.options;
         }
         switch (feature.properties.type) {
           case 'marker':
+            if (!options) {
+              options = model.get('marker')?.markerStyle;
+            }
             return new L.Marker(latlng, options);
           case 'circle':
```

Comment:
```suggestion
          case 'circle':
          	if (!options) {
              options = model.get('circle')?.pathOptions;
            }
```

When i was testing it, I got options `undefined` everytime, so let's add a check like other cases?